api/app.go

```diff
@@ -131,13 +133,20 @@ func (app *App) configureJaeger() {
 		zap.String("operation", "configureJaeger"),
 	)
 
+	disabled := os.Getenv("JAEGER_DISABLED") == "true"
```

Comment:
Viper already handles env var parsing, you just need to access the config entry. I'd have the following in the config yaml files:

```yaml
jaeger:
  disabled: true
  samplingProbability: 0.1
```

these values can be overridden by the PODIUM_JAEGER_DISABLED and PODIUM_JAEGER_SAMPLINGPROBABILITY env vars and will be retrieved like this 

```go
disabled := app.Config.GetBool("jaeger.disabled")
disabled := app.Config.GetFloat("jaeger.samplingProbability")
```